<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quicky_toolbar</class>
 <widget class="QToolBar" name="quicky_toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomToolBarArea|Qt::RightToolBarArea</set>
  </property>
  <action name="newNoteBtn">
   <property name="text">
    <string>+</string>
   </property>
   <property name="toolTip">
    <string>New Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="deleteNoteBtn">
   <property name="text">
    <string>-</string>
   </property>
   <property name="toolTip">
    <string>Delete Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="bgColorBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/color_icon.png</normaloff>:/icons/color_icon.png</iconset>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>Background Color</string>
   </property>
  </action>
  <action name="actionBoldBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalicBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderlineBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <addaction name="newNoteBtn"/>
  <addaction name="deleteNoteBtn"/>
  <addaction name="bgColorBtn"/>
  <addaction name="actionBoldBtn"/>
  <addaction name="actionItalicBtn"/>
  <addaction name="actionUnderlineBtn"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
